File: desafio-02/jldurigan/python/primos.py
stringNumeros = ""

for i in range(1, 10000):
    primo = True
    j = 1

    while j <= i and primo:
        if i != j and i % j == 0 and j != 1:
            primo = False

        j += 1

    if primo:
        stringNumeros += str(i) + ", "

stringNumeros = stringNumeros.rstrip(", ")

print(stringNumeros)

File: desafio-02/bloomingmiracle/c/primos.c
#include <stdio.h>

int main() {

    int n = 10000;

    int primo[10001];

    for (int i = 0; i <= n; i++) {
        primo[i] = 1;
    }

    primo[0] = 0;
    primo[1] = 0;

    for (int i = 2; i * i <= n; i++) {

        if (primo[i] == 1) {

            for (int j = i * i; j <= n; j += i) {
                primo[j] = 0;
            }
        }
    }

    for (int i = 2; i <= n; i++) {

        if (primo[i] == 1) {
            printf("%d\n", i);
        }
    }

    return 0;
}
